docs/reference/commandline/dockerd.md

```diff
@@ -1279,3 +1282,25 @@ $ sudo dockerd \
         --data-root=/var/lib/docker-bootstrap \
         --exec-root=/var/run/docker-bootstrap
 ```
+
+### Default network options
+
+The `--default-network-opt` flag, and the equivalent `default-network-opts` key
+in the configuratio file, lets you specify custom default options for custom
+networks.
+
+This option accepts a value with the format `driver=opt=value`, for example:
+
+```console
+$ sudo dockerd \
+  --default-network-opt bridge=com.docker.network.bridge.host_binding_ipv4=127.0.0.1 \
+  --default-network-opt bridge=com.docker.network.bridge.mtu=1234
+```
+
+New networks that you create inherit the specified options by default.
```

Comment:
"by default" is a bit confusing here (when are they _not_ inherited?); perhaps "as default" would work? 

I was slightly wondering as well if "inherit" could be interpreted incorrectly, as it feels like "dynamically inherit options from the default"; which would be incorrect, because the defaults are "baked" into the newly created network. We have the same "issue" for logging-driver and logging-driver options; I see for those we have a "note" https://docs.docker.com/config/containers/logging/configure/#configure-the-default-logging-driver

> Changing the default logging driver or logging driver options in the daemon configuration only affects containers that are created after the configuration is changed. Existing containers retain the logging driver options that were used when they were created. 


But honestly that's also a bit awkward; perhaps we can find a better way to make it clear from the start that "these are used as defaults when creating new networks" (and make clear that they are "baked in").